web-components/src/components/input-file/InputFile.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) Cisco Systems, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import reset from "@/wc_scss/reset.scss";
+import { customElement, html, LitElement, query, property, internalProperty } from "lit-element";
+import styles from "./scss/module.scss";
+import "@/components/button/Button";
+
+import { nothing } from "lit-html";
+@customElement("md-input-file")
+export class InputFile extends LitElement {
+  @property({ type: String }) getLocalization = "Get";
```

Comment:
Remember how you had file extensions allowed explicitly when you had this in code editor? i think i would be helpful to also have a way to limit file types here also if required (through a prop)

Reply:
Updated